stopify-continuations/src/runtime/eagerRuntime.ts

```diff
@@ -10,17 +10,20 @@ export class EagerRuntime extends common.Runtime {
     this.eagerStack = [];
   }
 
-  captureCC(f: (k: any) => any) {
+  captureCC(f: (k: (x: Result) => any) => any): any {
     this.capturing = true;
     throw new common.Capture(f, [...this.eagerStack]);
   }
 
   makeCont(stack: common.Stack) {
-    return (v: any, err: any=this.noErrorProvided) => {
```

Comment:
@baxtersa this change to the interface of continuations may break your work. So, I definitely don't want to merge it unless you confirm that it's okay. It is completely optional for what we are doing now.

Reply:
This if fine.